src/generateDapp.ts

```diff
@@ -97,6 +97,9 @@ export async function generateDapp(selection: Selections) {
       case "token-minting-dapp-template":
         await generateEnvFile(`VITE_FA_CREATOR_ADDRESS=""`);
         break;
+      case "token-staking-dapp-template":
+        await generateEnvFile(`VITE_FA_ADDRESS=""\nVITE_CREATOR_ADDRESS=""`);
```

Comment:
can we rename CREATOR_ADDRESS to REWARD_CREATOR_ADDRESS? just to make it more clear